app/services/location_services.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module LocationServices
+  class LocationServicesGeoJson
```

Comment:
As the class is namescoped under LocationServices, I would name it only "GeoJson". It's a little bit less verbose when you call it and you loose no information. `LocationServices::GeoJson`

Reply:
Good point, updating it.